pkg/serverless/logs/logs_test.go

```diff
@@ -471,7 +471,7 @@ func TestProcessLogMessageNoStringRecordFunctionLog(t *testing.T) {
 
 	logMessages := []logMessage{
 		{
-			logType: logTypeFunction,
+			stringRecord: "hi, log 2",
```

Comment:
The name of this test contains `NoStringRecord` but it looks like you added a `stringRecord` to the test data here. What was the motivation for this change? Also, is this related to adding Java integration tests?

Reply:
Per convo, this might be related to the fact that we haven't merged the v16 changes back into the main branch yet -- I'll wait for that to be merged before reviewing further.